core/src/test/java/io/confluent/kafka/schemaregistry/avro/AvroCompatibilityTest.java

```diff
@@ -83,7 +104,7 @@ public void testBasicBackwardsCompatibility() {
                checker.isCompatible(schema2, Collections.singletonList(schema1)));
     assertFalse("adding a field w/o default is not a backward compatible change",
                 checker.isCompatible(schema3, Collections.singletonList(schema1)));
-    assertFalse("changing field name is not a backward compatible change",
+    assertTrue("changing field name with an alias is a backward compatible change",
```

Comment:
What did aliasing have to do with this ?

Reply:
I think the previous `ResolvingGrammarGenerator` implementation handled aliases improperly. I changed this existing test case since aliasing a field is a compatible evolution.